django/conf/global_settings.py

```diff
@@ -543,6 +543,8 @@ def gettext_noop(s):
 CSRF_COOKIE_PATH = '/'
 CSRF_COOKIE_SECURE = False
 CSRF_COOKIE_HTTPONLY = False
+CSRF_USE_SESSIONS = False
+CSRF_SESSION_KEY = 'csrftoken'
```

Comment:
Please remove it and put it into a module level variable in the csrf module (and start it with an underscore) -- I really dislike settings, especially when it comes to security ;)